lib/types.ts

```diff
@@ -138,9 +124,7 @@ export type MarkRequired<T, RK extends keyof T> = Exclude<T, RK> & Required<Pick
 export type MarkOptional<T, K extends keyof T> = Omit<T, K> & Partial<Pick<T, K>>;
 
 /** Convert union type to intersection #darkmagic */
-export type UnionToIntersection<U> = (U extends any ? (k: U) => void : never) extends ((k: infer I) => void)
-  ? I
-  : never;
+export type UnionToIntersection<U> = (U extends any ? (k: U) => void : never) extends (k: infer I) => void ? I : never;
```

Comment:
pls undo this formatting change, even thought the brackets and newlines are not explicitly necessary,  they did considerably improve readability of this complex expression

Reply:
I think this is breaking change in prettier so I don't think that we can do anything about it...

Reply:
Yes, the format is done by prettier.

Reply:
:( that's why I "really don't like" it, but no choice here then

Reply:
> OK BOOMER 💥 

😉